libs/s25main/ingameWindows/iwStatistics.cpp

```diff
@@ -263,15 +275,16 @@ void iwStatistics::Draw_()
             continue;
 
         if(activePlayers[i])
+        {
             DrawRectangle(Rect(drawPt, Extent(34, 12)), player.color);
+
+            std::string playerId = std::to_string(player.GetPlayerId() + 1);
```

Comment:
It makes sense to do that only when actually required. So either:

1. replace `playerId` by this value directly inline
2. make this a variable containing the whole logic to move it out the `Draw` call
3. introduce a static function (local to this file) or a lambda (local to this function) returning the text to show given a `player` which might be easier to read as you can use if-elseif-else

Also make the variable `const` if you can